hotel-management/src/styles/components/button.css

```diff
@@ -0,0 +1,14 @@
+.btn {
+  display: inline-block;
+  border-radius: var(--border-radius-md);
+  border: none;
```

Comment:
Where cursor: pointer for clickable element

Reply:
I find that the `cursor: pointer` is default for tag `a` and `button`, so no need to define it in css